ui/pages/confirm-transaction-base/transaction-alerts/transaction-alerts.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+import { INSUFFICIENT_FUNDS_ERROR_KEY } from '../../../helpers/constants/error-keys';
+import { submittedPendingTransactionsSelector } from '../../../selectors/transactions';
+import { useGasFeeContext } from '../../../contexts/gasFee';
+import ActionableMessage from '../../../components/ui/actionable-message/actionable-message';
+import ErrorMessage from '../../../components/ui/error-message';
+import I18nValue from '../../../components/ui/i18n-value';
+
+const TransactionAlerts = () => {
+  const { balanceError, estimateUsed } = useGasFeeContext();
+  const pendingTransactions = useSelector(submittedPendingTransactionsSelector);
+
+  return (
+    <div className="transaction-alerts">
+      {pendingTransactions?.length > 0 && (
```

Comment:
What do you think about having `submittedPendingTransactionsSelector` always return some kind of an array, even if it is empty? This would avoid the null check here.

Reply:
I would avoid changing existing code at this point.